### PR TITLE
Add Zig Support for JetBrains IDEs

### PR DESCRIPTION
**Pull request by MarioAriasC**
Add Zig Support

### CODE REVIEW

**Comment by catdevnull** [comment]
How does it differ from [ice1000/intellij-zig](https://github.com/ice1000/intellij-zig)?

File: readme.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ A general-purpose programming language and toolchain for maintaining robust, opt
 
 
 ## Editor plugins
+* [MarioAriasC/zig-support](https://github.com/MarioAriasC/zig-support) Zig language support for JetBrains IDEs (IntelliJ, CLion and others)
```
Comment (catdevnull) [inline readme.md:96]:
```suggestion
* [MarioAriasC/zig-support](https://github.com/MarioAriasC/zig-support) - Zig language support for JetBrains IDEs (IntelliJ, CLion and others)
```

Comment (catdevnull) [inline readme.md:96]: It's redundant to mention Zig.

```suggestion
* [MarioAriasC/zig-support](https://github.com/MarioAriasC/zig-support) Language support for JetBrains IDEs (IntelliJ, CLion and others)
```

**Comment by MarioAriasC** [comment]
[ice1000/intellij-zig](https://github.com/ice1000/intellij-zig) Was declared dead by its maintainer more than two years ago [#32](https://github.com/ice1000/intellij-zig/issues/32) 

My plugin was updated today, and its in active development

File: readme.md [hunk 2]
```diff
@@ -93,6 +93,7 @@ A general-purpose programming language and toolchain for maintaining robust, opt
 
 
 ## Editor plugins
+* [MarioAriasC/zig-support](https://github.com/MarioAriasC/zig-support) - Language support for JetBrains IDEs (IntelliJ, CLion and others)
 * [ice1000/intellij-zig](https://github.com/ice1000/intellij-zig) - The IntelliJ IDEA plugin for the Zig programming language.
```
Comment (catdevnull) [inline readme.md:97]:
```suggestion
```

**Comment by catdevnull** [comment]
Thank you for your contribution!

**Pull request merged**